db/schema.rb

```diff
@@ -267,7 +267,7 @@
     t.boolean "passporting_benefit"
     t.string "benefit_type"
     t.string "has_benefit_evidence"
-    t.index ["type", "crime_application_id"], name: "index_people_on_type_and_crime_application_id", unique: true
+    t.index ["crime_application_id"], name: "index_people_on_crime_application_id", unique: true
```

Comment:
I'm also getting these diffs in my pr.. i think the schema file in main is not in sync with migrations somehow 🤔